uperf_wrapper/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM registry.access.redhat.com/ubi8:latest
 
 RUN dnf copr enable ndokos/pbench -y
-RUN dnf install -y --nodocs git python3-pip python3-numpy python3-requests python3-numpy pbench-uperf
+RUN dnf install -y --nodocs git python3-pip python3-requests pbench-uperf
```

Comment:
I guess we don't even need to install requests through rpm as it's in requirements.txt - https://github.com/cloud-bulldozer/snafu/blob/master/requirements.txt#L6

Reply:
We're also installing other dependencies through rpm in other tests. How do we should install them?. IMO installing them as RPM whenever is possible is safest.

Reply:
as we're getting to close to making snafu as a package, it might make sense to move towards using pip as package manager for consistency imo.